### PR TITLE
VACMS-8602: use block structure for slack message.

### PR DESCRIPTION
**Pull request by timcosgrove**
## Description
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8602

### CODE REVIEW

File: .github/workflows/content-release.yml [hunk 1]
```diff
@@ -309,32 +313,6 @@ jobs:
         if: ${{ steps.get-broken-link-info.outputs.UPLOAD_AND_NOTIFY == '1' && always() }}
         run: aws s3 cp ./logs/${{ env.BUILDTYPE }}-broken-links.json s3://vetsgov-website-builds-s3-upload/broken-link-reports/${{ env.BUILDTYPE }}-broken-links.json --acl public-read --region us-gov-west-1
 
-      - name: Notify VFS Platform Builds channel about broken links
```
Comment (timcosgrove) [inline .github/workflows/content-release.yml:312]: Decided it was time to only report broken links to one channel.

File: .github/workflows/content-release.yml [hunk 2]
```diff
@@ -309,32 +313,6 @@ jobs:
...
-        if: ${{ steps.get-broken-link-info.outputs.UPLOAD_AND_NOTIFY == '1' && always() }}
-        continue-on-error: true
-        env:
-          SSL_CERT_DIR: /etc/ssl/certs
-          SLACK_BOT_TOKEN: ${{ env.SLACK_BOT_TOKEN }}
-        with:
-          payload: ${{ steps.get-broken-link-info.outputs.SLACK_ATTACHMENTS }}
-          channel-id: ${{ env.VFS_PLATFORM_BUILDS_SLACK }}
-
-      - name: Output broken links info (fallback)
```
Comment (timcosgrove) [inline .github/workflows/content-release.yml:324]: removed the fallbacks as they weren't firing.

**Review by timwright12 — APPROVED** [review]

File: script/github-actions/check-broken-links-blocks.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-console */
+// https://cms-nc3gdbj3c2p9pizhf1sm8czjvtfeg1ik.demo.cms.va.gov -- Tugboat CMS with 2 broken links
```
Comment (timcosgrove) [inline script/github-actions/check-broken-links-blocks.js:2]: This file is copied from https://github.com/department-of-veterans-affairs/content-build/blob/master/script/github-actions/check-broken-links.js and differs mainly in how the payload is constructed. Switched from an `attachment` structure to `blocks`, per Github's current recommendations. https://api.slack.com/messaging/composing/layouts#when-to-use-attachments

File: src/site/stages/build/plugins/modify-dom/check-broken-links/index.js [hunk 4]
```diff
@@ -91,6 +91,7 @@ module.exports = {
       summary: markdownSummary,
       isHomepageBroken,
       brokenLinksCount,
+      brokenPages,
```
Comment (timcosgrove) [inline src/site/stages/build/plugins/modify-dom/check-broken-links/index.js:94]: This just adds the structured Javascript object back to the file that gets output.

File: script/github-actions/check-broken-links-blocks.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    text: {
+      type: 'mrkdwn',
+      text: `${icon} *<!subteam^S010U41C30V|cms-helpdesk> ${brokenLinks.brokenLinksCount} broken links found in ${envName}*\n\n Workflow run: <${SERVER_URL}>`,
+    },
+  });
+  const linkBlocks = brokenLinks.brokenPages.map((page, idx) => {
+    const problemMarkup = page.linkErrors.map(error => {
+      const destination =
+        error.target.substring(0, 1) === '/'
+          ? `https://va.gov${error.target}`
```
Comment (timcosgrove) [inline script/github-actions/check-broken-links-blocks.js:36]: Need to account for environments here.

**Review by cweagans — APPROVED** [review]
I would recommend getting rid of the `new` suffix everywhere and just replace what's currently there, but otherwise LGTM!

**Pull request merged**